README.md

```diff
@@ -32,3 +62,9 @@ image = bf_import("file.msr")
 ```
 
 The variable `image` will contain an array of ImageMetadata, the data will be the type that the format specifies: __Int8__, __UInt8__, __Int16__, __UInt16__, __Int32__, __UInt32__, __Float32__, __Float64__ or __Bool__.
+
+If you just want a plain array of the first frame:
+
+```julia
+plain_array = collect(image[1].data)
```

Comment:
We can keep this here if you think people find the views confusing or there is some other reason to make the copy but use the `arraydata` function instead of accessing the `.data` property. Maybe also use `copy` instead of `collect`, not much of a difference but makes it a bit more clear that you are copying a view and not collecting something like a generator.
```suggestion
plain_array = copy(arraydata(image[1]))
```

Reply:
Sure. This makes sense.